### PR TITLE
archive index.html for feb 2021 meeting; begin changes for july 2021 …

### PR DESCRIPTION
**Pull request by TomGlanzman**
…meeting

### CODE REVIEW

**Comment by TomGlanzman** [comment]
This pre-meeting-checklist is ready to go for the July 2021 DESC meeting.

File: index.html [hunk 1]
```diff
@@ -119,8 +119,8 @@ <h2>Access meeting resources</h2>
           </div>
           <div class="text">
                Check out the event calendar on the
-               <a href="https://confluence.slac.stanford.edu/x/ypzgEQ"
-               title="link to Confluence">Meeting Schedule</a> page.
+               <a href="https://teamup.com/ksv93cghyu9t4nn8az"
+               title="link to Confluence">Meeting Agenda</a> page.
```
Comment (yymao) [inline index.html]: The link goes to the calendar directly, not the confluence agenda page. 
```suggestion
               Check out the <a href="https://teamup.com/ksv93cghyu9t4nn8az"
               title="link to TeamUp Calendar">event calendar</a>. 
```

**Review by yymao — APPROVED** [review]
Thanks @TomGlanzman 

I made a minor comment regarding the link to meeting agenda page. 

Also, Zoom now has a new feature for users to add their pronouns: https://support.zoom.us/hc/en-us/articles/4402698027533-Adding-and-sharing-your-pronouns

Otherwise looks good to me; pre-approving. 

Once this is merged, I'll update the served branch.

File: index.html (hunk 1 above)
Comment (TomGlanzman) [reply index.html]: How's that?

**Comment by yymao** [comment]
LGTM! Thanks!

**Pull request merged**